apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -92,6 +92,32 @@ $tokensTable-item-min-height: 128px;
     }
 }
 
+.c-tokensTable-example-radius-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: f.$color-background-subtle;
+    min-height: $tokensTable-spacing-container-height;
+
+    @include f.media('>=wide') {
+        display: block;
+        background-color: transparent;
```

Comment:
I think we might need to change the min-height for wide screens as it doesn't really match the spacing we have in designs (80px). 

<img width="737" alt="Screenshot 2023-01-13 at 09 44 44" src="https://user-images.githubusercontent.com/28605803/212277216-591f6011-17f2-4c0d-a2fd-1321d004c64d.png">